cpp/src/copying/shift.cu

```diff
@@ -19,6 +19,7 @@
 #include <cudf/detail/copy.hpp>
 #include <cudf/detail/nvtx/ranges.hpp>
 #include <cudf/detail/valid_if.cuh>
+#include <cudf/fixed_point/fixed_point.hpp>
```

Comment:
This is not used in this file in this PR. I would not add it until you need it.

Reply:
Done.